package.json

```diff
@@ -514,17 +514,17 @@
                   "items": {
                     "type": "string"
                   },
-                  "description": "Array of symbol server URLs (example: http\u200b://MyExampleSymbolServer) or directories (example: /build/symbols) to search for .pdb files. These directories will be searched in addition to the default locations -- next to the module and the path where the pdb was originally dropped to.",
```

Comment:
Undo removing `\u200b`

We use '\u200b' (unicode zero-length space character) to break VS Code's URL detection regex for URLs that are examples.

Reply:
I'll fix it by running the update dependencies task.

Reply:
Fixed